packages/app/src/files-manager/components/files-manager/files-manager.component.tsx

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import React from "react"
+import { FunctionComponent } from "Renderer/types/function-component.interface"
+import { FilesManagerContainer } from "App/files-manager/components/files-manager/files-manager.styled"
+import FilesSummary from "App/files-manager/components/files-summary/files-summary.component"
+import { Type } from "Renderer/components/core/icon/icon.config"
+import { FilesManagerTestIds } from "App/files-manager/components/files-manager/files-manager-test-ids.enum"
+
+export interface DiskSpaceCategory {
+  filesType: string
+  occupiedMemory?: number
```

Comment:
`megabyteSize` or `totalMegabyteSize` to mark measure type (MB)? 🤔

Reply:
Changed in [998d8d6](https://github.com/mudita/mudita-center/pull/598/commits/998d8d6fdea8e48ae9544f8aa93d79c6152e1f84)